jdaviz/configs/default/plugins/markers/tests/test_markers_plugin.py

```diff
@@ -0,0 +1,240 @@
+from numpy.testing import assert_allclose
+
+from jdaviz.core.marks import MarkersMark
+from jdaviz.configs.imviz.tests.utils import BaseImviz_WCS_NoWCS
+
+
+def _get_markers_from_viewer(viewer):
+    return [m for m in viewer.figure.marks if isinstance(m, MarkersMark)][0]
```

Comment:
Feels weird to grab the whole list only to keep the first element. But I also don't know if the suggestion is any better. 🤷 
```suggestion
    markers = None
    for m in viewer.figure.marks:
        if isinstance(m, MarkersMark):
            markers = m
            break
    return markers
```

Reply:
Yeah... it felt a bit ugly to me too.  We could probably also use `next`, but it's in a test, not in the code, so I don't think it's worth profiling to see which is fastest.  Which do you think is clearest to read?

Reply:
I would naturally prefer my own suggestion. 🤣  Better ask a neutral party! 😉

Reply:
I agree that it probably doesn't matter. That said, I'm a fan of:
```
[unpacked_single_item] = [m for m in ...]
return unpacked_single_item
```
because I like brackets everywhere.